.github/workflows/ci-template-fast-api-check-spec.yml

```
@@ -1,4 +1,4 @@
-name: CI - FastAPI Spec Check
+name: CI - FastAPI Template Check Spec
```

```suggestion
name: CI - Template: FastAPI Check Spec
```

To group with other templates and match order of files

---

Also would we want to just add the `check spec` as a new job or step in `ci-template-fast-api.yaml`?